Uppsala/UppEt9.xml

```diff
@@ -0,0 +1,323 @@
+                            
+                            <handNote xml:id="h2" script="Ethiopic" corresp="#a1">
+                                <desc>Clear hand (<foreign xml:lang="de">deutlich</foreign>), 
+                                    smaller and later than <ref type="hand" corresp="#h1"/>
+                                    (<bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">47</citedRange></bibl>).</desc>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="1v" to="4"/>
+                                    <locus target="#172v"/>
+                                    <desc type="GuestText"><title type="complete" ref="LIT7328MalkeaYostos"/>
+                                    </desc>
+                                </item>
+                            </list>
+                            <list>
+                                <item xml:id="a2">
+                                    <desc type="Unclear">
+                                        <bibl>Without specifying on which folio,
+                                            <ptr target="bm:Zettersteen1899Upsala"/><citedRange unit="page">519</citedRange></bibl>
+                                        notes that the names of <persName ref="PRS14760KenfaGabreel"/> and of 
+                                        his son <persName ref="PRS14761TaklaHaymanot"/>
+                                        are written underneath <ref type="item" corresp="#a1"/>.
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards"><material key="wood">Wooden</material> boards.
+                                </decoNote>
+                                <decoNote xml:id="b2" type="Cover">The boards are covered with 
+                                    blind-tooled <material key="leather"/>.
+                                </decoNote>
+                                <decoNote xml:id="b3" type="SlipCase">Preserved in a <foreign xml:lang="gez">māḫdar</foreign>.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                        
+                    </physDesc>
+                    
+                    <history>
+                        <origin>
+                            <origDate notBefore="1700" notAfter="1899">18th–19th century</origDate>
+                        </origin>
+                        <provenance>The name of the original owner has been erased and substituted
```

Comment:
This information should also be given as an extra item ,see https://betamasaheft.eu/Guidelines/?id=additionsVaria
But I think it's very useful to repeat it in provenance

Reply:
The guidelines seem to suggest that this CAN be done, not that it has to be done. Or am I reading it wrong?

Reply:
Yes, I guess it makes sense both in extras (when it's based on notes/supplications in the manuscript) and in history